timescale-cloud/index.md

```diff
@@ -1,38 +1,74 @@
 # Timescale Cloud
+Managed services for TimescaleDB (MST) provides managed TimescaleDB services
+hosted in more than 75 regions using AWS, Azure, or GCP. You can use MST to
+create database instances, or services, in the cloud and automate most of your
+most common operational tasks. This allows you to spend more time focusing on
+your time-series workloads and less time worrying about database management.
 
-Timescale Cloud is a managed TimescaleDB service hosted in 75+ regions in AWS, 
-Azure, or GCP. You can [try Timescale Cloud for free][sign-up], no credit card 
-required.
+*   Find out [about services][create-a-service]
+*   Create a [multi-node cluster][cloud-multi-node]
+*   View [service logs][viewing-service-logs]
+*   Use [VPC peering][vpc-peering]
+*   Understand [security for MST][security]
 
-### Hosted TimescaleDB
-Built and operated by the team behind TimescaleDB, Timescale Cloud provides a 
+<highlight type="note">
+Managed service for TimescaleDB (MST) used to be called _Timescale Cloud_.
+</highlight>
+
+Some important MST terms:
+*   Account: Your MST account. You can register for an account on the
+    [MST signup][sign-up] page.
```

Comment:
```suggestion
Some important Timescale Cloud terms:
*   **Account:** Your Timescale Cloud account. You can register for an account on the
    [Timescale Cloud signup][sign-up] page.
```

Reply:
Do you want bolding on all the terms? It annoys me that we can't do a 'proper' definition list.